tritium-registry/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply from: "${rootDir}/gradle/publish-jars.gradle"
+
+dependencies {
+    processor 'com.google.auto.service:auto-service'
+    processor 'org.immutables:value'
+
+    compile 'com.palantir.safe-logging:safe-logging'
+    compile 'io.dropwizard.metrics:metrics-core'
```

Comment:
this brings in the same dependencies as before, no?

Reply:
this only brings in the Metrics (Counter, Timer, etc), not anything else from tritium (tritium-api, tritium-core)
```
compile - Dependencies for source set 'main' (deprecated, use 'implementation ' instead).
+--- com.palantir.safe-logging:safe-logging: -> 0.1.3
\--- io.dropwizard.metrics:metrics-core: -> 3.2.5
     \--- org.slf4j:slf4j-api:1.7.22 -> 1.7.12
```

Reply:
ok

Reply:
sorry, didn't right properly